Code/Framework/AzNetworking/AzNetworking/AzNetworkingConfiguration.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/PlatformDef.h>
+
+#if defined(_LIB)
```

Comment:
This should be `#if defined(AZ_MONOLITHIC_BUILD)`

Tagging @burelc-amzn,@jeremyong-az and @AMZN-koppersr for visibility, since this is prototyping the SHARED library approach

Reply:
Got it, for context I patterned this off SceneUI.